torch/nn/modules/module.py

```diff
@@ -47,6 +47,30 @@ def forward(self, x):
 
     dump_patches = False
 
+    r"""This allows better BC support for :meth:`load_state_dict`. In
+    :meth:`state_dict`, the version number will be saved as in the attribute
+    `_versions` of the returned state dict, and thus pickled. `_versions` is a
+    dictionary. Its keys follow the naming convention of state dict, but ends
+    with key ``"version"`` and is associated with the version number of that
```

Comment:
nit: we could probably skip the `.version` suffix because it's unnecessary. Otherwise, if we anticipate that we might want to stash some extra metadata in the future, then we could rename `._versions` to `._metadata`